### ISSUE TITLE
Clarification on Definitive Edition

### ISSUE THREAD

**Issue by SoonDead** [beatcracker/Ori-Controller-Remap#4]
I'm trying to play Ori and The Blind Forest Definitive Edition with a Logitech Dual Action gamepad.

This tool is pretty high on the list if you search for "ori and the blind forest joystick remap" so I tried it and managed to remap mostly everything except the right trigger, like in #3 .

Then I searched around for more info as remapping with joytokey did not work.

I found out that the definitive edition has [two textfiles](http://steamcommunity.com/app/387290/discussions/0/364040166669190939/) to remap my controller buttons. With this I was able to remap all my buttons without problems (reverting Assembly-CSharp.dll before of course). I used the numeric values provided in Ori_Controller_Remap.ini .

This should be in the readme.

TL;DR: The definitive edition does not require you to use this tool, and has built in remap support.

**Comment by beatcracker** [comment]
Thanks, fixed.

**Comment by SoonDead** [comment]
Wow, that was fast, thank you.
